hathor/manager.py

```diff
@@ -290,37 +288,29 @@ def _initialize_components(self) -> None:
             assert tx.hash is not None
 
             t2 = time.time()
-            if t2 - t1 > 5:
+            dt = t2 - t1
+            dcnt = cnt - cnt2
+            if dt > 30:
                 ts_date = datetime.datetime.fromtimestamp(self.tx_storage.latest_timestamp)
-                self.log.info(
-                    'Verifying transations in storage... avg={avg:.4f} tx/s total={total} (latest timedate: {ts})',
-                    avg=(cnt - cnt2) / (t2 - t1),
-                    total=cnt,
-                    ts=ts_date,
-                )
+                self.log.info('load and validate transactions...', tx_rate=dcnt / dt, tx_new=dcnt, dt=dt,
+                              total=cnt, latest_ts=ts_date)
                 t1 = t2
                 cnt2 = cnt
             cnt += 1
 
             try:
                 assert self.on_new_tx(tx, quiet=True, fails_silently=False)
             except (InvalidNewTransaction, TxValidationError):
-                pretty_json = json.dumps(tx.to_json(), indent=4)
-                self.log.error('An unexpected error occurred when initializing {tx.hash_hex}\n'
-                               '{pretty_json}', tx=tx, pretty_json=pretty_json)
+                self.log.error('unexpected error when initializing', tx=tx, exc_info=True)
                 raise
 
             if time.time() - t2 > 1:
-                self.log.warn('Warning: {} took {} seconds to be processed.'.format(tx.hash.hex(), time.time() - t2))
+                dt = time.time() - t2
+                self.log.warn('tx took too long.', tx=tx.hash_hex, dt=dt)
 
         # self.stop_profiler(save_to='profiles/initializing.prof')
         self.state = self.NodeState.READY
-        self.log.info(
-            'Node successfully initialized (total={total}, avg={avg:.2f} tx/s in {dt} seconds).',
-            total=cnt,
-            avg=cnt / (t2 - t0),
-            dt=t2 - t0,
-        )
+        self.log.info('ready', tx_count=cnt, tx_rate=cnt / (t2 - t0), total_dt=t2 - t0)
```

Comment:
I am still not sure why change 'Node successfully initialized' to 'ready'. Right now we have some things that could be ready, e.g. prometheus, metrics, stratum. I like being verbose in this case

Reply:
It's scoped by `hathor.manager`, so in my head this was fine.

Reply:
Ah, right. I forgot about the scope, then it's fine.

As you said before, I think an image of the new logs would be good.